### PR TITLE
re-formatting aquifer files with clang-format

### PR DESCRIPTION
**Pull request by GitPaean**
It will make the editing of these files easier.

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
I think `clang-format` does a pretty good job.

File: opm/simulators/aquifers/AquiferCarterTracy.hpp [hunk 1]
```diff
@@ -31,249 +31,250 @@
...
                 }
             }
+            auto cellCenter = grid.getCellCenter(Base::cell_idx_.at(idx));
+            Base::cell_depth_.at(idx) = cellCenter[2];
+        }
+
+        const double eps_sqrt = std::sqrt(std::numeric_limits<double>::epsilon());
+        for (size_t idx = 0; idx < Base::cell_idx_.size(); ++idx) {
+            Base::alphai_.at(idx) = (denom_face_areas < eps_sqrt)
+                ? // Prevent no connection NaNs due to division by zero
```
Comment (GitPaean) [inline opm/simulators/aquifers/AquiferCarterTracy.hpp]: The breaking of this is arguable, while the original form is not good either.

File: opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp [hunk 2]
```diff
@@ -1,225 +1,208 @@
...
+}
+
+template <typename TypeTag>
+void
+BlackoilAquiferModel<TypeTag>::initialSolutionApplied()
+{
+    if (aquiferCarterTracyActive()) {
+        for (auto aquifer = aquifers_CarterTracy.begin(); aquifer != aquifers_CarterTracy.end(); ++aquifer) {
+            aquifer->initialSolutionApplied();
+        }
```
Comment (bska) [inline opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp]: While you're already here, would you mind switching this (and the Fetkovich branch too) into using range-`for`?

Comment (GitPaean) [reply opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp]: Sure.  Should we remove the usage of `at`?

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by bska** [comment]
Merging now since Jenkins approves.

**Pull request merged**